modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,258 @@
++
+image:qnap/qnap-app_center.png[QNAP App Center with Container Station installed]
+
+== Installation of ownCloud
+
+In the App Center search for ownCloud and click btn:[Install]. Confirm that you want to install this app. During installation, you'll see a progress bar.
+
+After a successful installation, ownCloud will show up on your desktop and in the App Center under menu:QNAP Store[My Apps]. Click btn:[Open] under the ownCloud icon and proceed with the configuration.
+
+image:qnap/qnap_main_menu.png[QNAP Main Menu and ownCloud installed]
+
+== Configuration
+
+=== First Steps
+
+Log in as user `admin`. The default password is `admin` until you change it. In the upper right-hand corner, click on menu:admin[Settings]. In the left hand navigation bar, you find personal configuration options and administration-specific options. Under menu:Personal[General] you can change the password of your admin user.
+
+For more information on the web interface, refer to the xref:user_manual:webinterface.adoc[Web UI] documentation.
+
+=== Configure your Email.
+
+Email is an essential tool in ownCloud for notifying users, user registration, password resets, etc. Logged-in as admin, you can configure email in the menu:Admin[General] settings.
+
+For more information, refer to section xref:configuration/server/email_configuration.adoc[Email Configuration].
+
+=== Cron
+
+ownCloud performs regular tasks in the background. In order to schedule them, a mechanism called "cron" is used. It's already set up for you, so you don't need to do anything. Make sure not to change cron-related settings unless you have a very good reason to do so.
+
+NOTE: In general, we recommend to keep the default settings in the admin section.
+
+== Users
+
+A key feature of ownCloud is to allow users to share files. In the free community edition you can create up to five users, including an admin user, and grant access to an unlimited number of guest users.
+
+=== Prerequisites:
+
+. An email server must be configured and working.
+
+. The Guests app must be installed and enabled in menu:admin[Settings > Admin > Apps]. For detailed information on the Guests app, see the xref:configuration/user/guests_app.adoc[Guests App section].
++
+image:qnap/marketplace.png[ownCloud Marketplace and Guests App]
+
+. Guest users must be able to access ownCloud either from within the same local network or via the internet.
+
+=== Managing Users
+
+In the menu:admin[Users] section you can create, disable, enable or delete users, add them to groups, grant administrator privileges and more.
+
+image:qnap/users_on_qnap.png[User Management]
+
+For detailed information see xref:configuration/user/user_configuration.adoc[User Management].
+
+After you have created users, they can upload files and share these with other ownCloud users and guest users. Folders can be created and shared as well. Sharing works via the email address of the user who should be granted access to certain files.
```

Comment:
"Sharing works via the email address of the user who should be granted access to certain files."

Can  you have "ordinary/real" users, who do not have an email address? For those "ordinary/real" users, the sharing should work by user name? I am a bit mixed up reading this section and working out which sentences are about Guest users, and which are about ordinary users.

Reply:
A valid email address needs to be specified for all users, but of course the user name is enough if you want to share files. Will clarify.